brain_games/scripts/brain_games.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python
 
 
+from brain_games.cli import welcome_user
+
+
 def main():
     print("Welcome to the Brain Games!")
```

Comment:
enhancement: it's a good practice to keep the things separately - in this case, you have `cli.py` file that is responsible for working with cli (prints, promts, etc). please, move 
```python
print("Welcome to the Brain Games!")
```
to some function in `cli.py`, e. g. `welcome` and use it here